nrrd-design-system/components/components/links/README.md

```diff
@@ -1,3 +1,14 @@
 # Links
 
-*Self explanatory?*
+We use a variety of link styles depending on function and context. We use at least two visual characteristics in combination to denote links (e.g., color _and_ underline or color _and_ heavier weight).
+
+* links with relatively heavy font weight (400), underline on hover
```

Comment:
This is super helpful guidance to have, thanks for adding!

Since Fractal's default display of markdown is....clunky, would you consider using only one level of bullets here for better readability?

**Something like this uses `<h4>` style headers and list items**
![screen shot 2018-04-06 at 3 31 21 pm](https://user-images.githubusercontent.com/11636908/38440545-cc7a4a14-39af-11e8-809b-66ca805a9579.png)

**Before, with nested bullets**
![screen shot 2018-04-06 at 3 24 14 pm](https://user-images.githubusercontent.com/11636908/38440579-df76b8b4-39af-11e8-9490-ca5af9e5c143.png)